src/views/questionView.js

```diff
@@ -36,7 +45,7 @@ export const createQuestionView = (props) => {
   const showAnswer = (currentQuestion, score) => {
     console.log({ currentQuestion }, score);
     //-y- if the answer is true, scoreDisplay is updated
-    scoreDisplay.innerHTML = score;
+    scoreDisplay.innerHTML = 'Score: ' + score;
```

Comment:
If you only need to set text then use `.textContent`: use the option that does the job.

```suggestion
    scoreDisplay.textContent= 'Score: ' + score;
```